hac.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: cloud.redhat.com/v1alpha1
+kind: Frontend
+metadata:
+  name: hac-core
+spec:
+  envName: env-boot
+  title: HAC
+  deploymentRepo: http://github.com/RedHatInsights/hac-core-frontend
+  API:
+    versions:
+      - v1
+  frontend:
+    paths:
+      - /apps/hac-core
+  image: quay.io/redhat-cloud-services/hac-core:ci-beta
+  navItems:
+    - appId: "hacCore"
+      title: "Core"
+      href: "/hac"
+    - appId: "hacDev"
+      title: "App Studio"
+      href: "/hac/app-studio"
```

There should also be a couple of `dynamicNav` NavItems.
```suggestion
  navItems:
    - appId: "hacCore"
      title: "Core"
      href: "/hac"
    - title: "Dynamic nav admin",
      dynamicNav: "hacCore/admin-demo-section"
    - title: "Dynamic nav",
      dynamicNav: "hacCore/dev-demo-section"
    - appId: "hacDev"
      title: "App Studio"
      href: "/hac/app-studio"
```

---

Ah I see now. Got it all squared away. Thanks!